README.md

```diff
@@ -1,9 +1,27 @@
 # Quasar MariaDB/MySQL Plugins [![Discord](https://img.shields.io/discord/373302030460125185.svg?logo=discord)](https://discord.gg/pSSqJrr)
 
+## Datasource
```

Comment:
Is there any additional work required to add support for MemSQL and MySQL? Below in the README, there is a connection configuration specified for only MariaDB. Will the MemSQL and MySQL configs follow the same config pattern, or do they have any specifics that warrant a distinct config?

Same question but from the context of the product: after this lands, will we be able to add datasource+destination tiles for each of: MariaDB, MemSQL, MySQL with no additional BE work (the only work would be the FE work to add the connector UI forms)?

Reply:
At the moment, they all use the same config and you just point the plugins at any of them. There are some minor differences in datatype representations (whether boolean values are represented as `BIT` or `TINYINT`, for example), otherwise they're operationally similar.

All the tests run and pass against all three vendors locally and against MariaDB and MySQL in CI, I tried to add MemSQL to CI, but it is giving some odd errors using the same configuration that works locally, there is a ticket to sort this out.

If it turns out after some actual usage that we need to handle certain things differently per-vendor, then we can do so, but there doesn't seem to be a need to yet.

Reply:
> will we be able to add datasource+destination tiles for each of: MariaDB, MemSQL, MySQL with no additional BE work

Yes